### ISSUE TITLE
Why is getChr deprecated?

### ISSUE THREAD

**Issue by jrobinso** [samtools/htsjdk#1201]
I get zillions of these warnings when compiling IGV

Warning:(241, 46) java: getChr() in htsjdk.tribble.Feature has been deprecated

Really, why is this neccessary?   I'm not going to change all these to getContig.   I realize not every sequence is a chromosome but not every sequence is a contig either.      I know this is old news but am raising it now in hopes that you would consider removing the @deprecated tag.   As the creator of tribble, maybe you can give me a break here?

**Comment by nh13** [comment]
For background: https://github.com/samtools/htsjdk/issues/197

**Comment by tfenne** [comment]
I'm sorta with Jim here.  I'd be more on-board with the deprecation if we'd gone with `getRefName` or similar, but as Jim points out `contig` is just as incorrect as `chr` - moreso if you consider that most of the time folks are aligning to genomes containing chromosomes that are scaffolds not contigs!  @lbergelson & @jacarey  thoughts?

**Comment by jrobinso** [comment]
Why not just leave it as is and have them be synonyms?   i.e. just remove @deprecated.

**Comment by jrobinso** [comment]
The problem I currently have is finding true deprecations in this sea of getChr warnings.

**Comment by lbergelson** [comment]
We wanted to encourage a single method name over a large variety of different similar method names by deprecating all but one of them.  I understand that `getContig` may not be the ideal name.  I believe we've settled on `getRefName` for htsjdk3.  I'd prefer to not have 2 un-deprecated versions though since it leads to confusion.

I've opened a PR https://github.com/igvteam/igv/pull/590 to migrate igv to `getContig`.   Maybe that can solve the problem?
